src/components/LoadingIndicator/LoadingIndicator.vue

```diff
@@ -30,8 +30,8 @@ export default {
 
 <style lang="scss" scoped>
 .loading-gif {
-  background: #fff url(https://s3-us-west-2.amazonaws.com/public.lob.com/sites/loading-gif.svg) no-repeat center;
-  background-size: 12px 12px;
+  background: #fff url(https://s3-us-west-2.amazonaws.com/public.lob.com/sites/spinner-medium.svg) no-repeat center;
+  background-size: 24px 24px;
```

Comment:
remind me, what are `animate-spin` and `${loadingClass}`?

Reply:
I mean, I read "tailwind class that spins it!", but I thought it already used to spin?

Reply:
apologies I should have explained this! the [old gif](https://s3.us-west-2.amazonaws.com/public.lob.com/sites/loading-gif.svg) used to spin on its own because it has keyframes in it.
The new one does not, I just picked the static design off figma - so we will use CSS to spin it (that way we may also change the timing in the future if we wish)
`animate-spin` is a simple 1s linear rotation keyframe ([docs here](https://tailwindcss.com/docs/animation))

`loadingClass` is a String prop that allows you to add extra classes to the spinner when the component is used (ie you can change the size). we have very rarely used this and I think we may remove it later when we implement different sizes of the spinner - but for now I am leaving it in as it is used in a couple spots on the dashboard.

Reply:
although now that I think of it, we should maybe put the keyframes in the svg, because of the way this is structured as a wrapper, it may cause problems otherwise! will try it

Reply:
thanks for explaining